python/equistore-torch/tests/operations/pow.py

```diff
@@ -0,0 +1,17 @@
+import torch
+
+import equistore.torch
+
+from .data import load_data
+
+
+def test_operation_as_python():
+    tensor = load_data("qm7-power-spectrum.npz")
+    power_tensor = equistore.torch.pow(tensor, 2.0)
+    assert equistore.torch.equal_metadata(power_tensor, tensor)
+
+
+def test_operation_as_torch_script():
+    tensor = load_data("qm7-power-spectrum.npz")
+    power_tensor = torch.jit.script(equistore.torch.pow)(tensor, 2.0)
+    assert equistore.torch.equal_metadata(power_tensor, tensor)
```

Comment:
should also check the values

Reply:
That's a fair point. We should probably decide a policy for these tests. Are they only supposed to check some very basic properties to ensure that the ouput isn't completely wrong or should they be proper tests? @Luthaf 
Worth deciding before writing more of these

Reply:
I think checking the output in one simple case would be nice and allow to catch some errors. We don't have to be as thorough as the main tests for corner cases and special values.

One extra thing that could be helpful in these tests would be to check the type of the output (see `remove_gradients`).